content/posts/ai_konferens_2024-02-07.md

```diff
@@ -0,0 +1,63 @@
+title: Summering av eventet AI i klimatets tjänst onsdagen 7 februari 2024
+date: 2024-02-08
+modified:
+author: Christoffer Strömberg
+category: Event
```

Comment:
Should it be under event or _Projekt_, as the other course I attended is labelled as _Projekt_.

Reply:
updated to Projekt.